### DISCUSSION TITLE
FMMT PEIM rebase feature Support

### DISCUSSION THREAD

**Discussion by AshrafAliS** [tianocore/edk2#6485] [General]
https://github.com/tianocore/edk2/tree/master/BaseTools/Source/Python/FMMT#known-issue

Checking when can we expect the FMMT Tool support for PEIM rebase FVs

**Comment by AshrafAliS** [comment]
https://bugzilla.tianocore.org/show_bug.cgi?id=4879

**Reply by mdkinney** [reply]
It is unassigned.  Can you work on proposed fix and submit a PR?

**Comment by AshrafAliS** [comment]
There were mainly 2 functions needs to be ported to Python
1. GetBaseAddrFromFsp
2. RebaseFfs

i have converted the 1st function GetBaseAddrFromFsp, 
RebaseFfs is bigger function, analyzing this function and functionality, post that i can work on it.

**Comment by AshrafAliS** [comment]
Can we get some support from tool team. those rebase functions are very huge

**Reply by makubacki** [reply]
> Can we get some support from tool team. those rebase functions are very huge

@AshrafAliS, I suggest you create an issue from this discussion, then it can be assigned to a tool team member.

**Comment by AshrafAliS** [comment]
Hi @makubacki 

Issue is already created: https://github.com/tianocore/edk2/issues/10552

**Reply by makubacki** [reply]
Thanks. It looks like it was dropped in the BZ to GH Issue migration. I've updated it so it will hopefully get attention during bug triage.
